clients/venice-push-job/src/test/java/com/linkedin/venice/hadoop/mapreduce/datawriter/reduce/TestVeniceReducer.java

```diff
@@ -71,15 +78,15 @@ public void testReducerPutWithTooLargeValueAndChunkingDisabled() {
     AbstractVeniceWriter mockWriter = mock(AbstractVeniceWriter.class);
     when(mockWriter.put(any(), any(), anyInt(), any(), any()))
         .thenThrow(new RecordTooLargeException("expected exception"));
-    testReduceWithTooLargeValueAndChunkingDisabled(mockWriter, setupJobConf());
+    testReduceWithTooLargeValueAndChunkingDisabled(mockWriter, setupJobConf(100));
```

Comment:
Is it ever not 100?

Reply:
Yeah, there is one test where the value is 1 and a few tests where this value is obtained from a `dataProvider` and the space of values there is `1, 10, 1000`